src/scss/component.scss

```diff
@@ -439,3 +439,26 @@
     opacity: 0;
   }
 }
+
+
+
+/*
+* netTrek
+* disable selection
+*/
+.mddtp-picker-date, .mddtp-picker-time
+{
+  -webkit-touch-callout: none; /* iOS Safari */
+  @include prefixer(user-select, none, webkit moz ms spec);
+}
+
+
+/*
+* netTrek
+* disable -webkit-tap-highlight-color for iOS
+*/
+.mddtp-picker-time {
+  .mddtp-picker__circularView, .mddtp-time__circularHolder {
```

Comment:
each selector is in a single line now but I cannot write the selectors it in lower case because U use it in camelcase :( See line 376 in JS src

```
 			// inside body
			this._addId(circularHolder, 'circularHolder')
			this._addClass(circularHolder, 'circularHolder')
			this._addId(needle, 'needle')
			needle.classList.add('mddtp-picker__selection')
			this._addClass(dot, 'dot')
			this._addClass(line, 'line')
			this._addId(circle, 'circle')
			this._addClass(circle, 'circle')
			this._addId(minuteView, 'minuteView')
			minuteView.classList.add('mddtp-picker__circularView')
			minuteView.classList.add('mddtp-picker__circularView--hidden')
			this._addId(fakeNeedle, 'fakeNeedle')
			fakeNeedle.classList.add('mddtp-picker__circle--fake')
			this._addId(hourView, 'hourView')
			hourView.classList.add('mddtp-picker__circularView')
```